tools/props/Versions.props

```diff
@@ -60,7 +61,7 @@
     <SystemDataOdbcVersion21>4.5.0</SystemDataOdbcVersion21>
     <SystemDataOdbcVersion>4.6.0</SystemDataOdbcVersion>
     <SystemNetSocketsVersion>4.3.0</SystemNetSocketsVersion>
-    <SystemIdentityModelTokensJwtVersion>5.6.0</SystemIdentityModelTokensJwtVersion>
+    <SystemIdentityModelTokensJwtVersion>6.8.0</SystemIdentityModelTokensJwtVersion>
```

Comment:
This had to be upgraded because starting from 2.1.0, we now use version 6.8.0 of `Microsoft.IdentityModel.Protocols.OpenIdConnect`, which uses version 6.8.0 of `
System.IdentityModel.Tokens.Jwt`, resulting in a package downgrade error at build.